### PR TITLE
[ruby/en] Adds Struct syntax for english Ruby document

### PR DESCRIPTION
**Pull request by pzula**
I thought it would be helpful for newcomers to see `Struct` used in Ruby, as it is often found in open source projects.

### CODE REVIEW

**Comment by Oire** [comment]
Looks good. @adambard, @levibostian, @vendethiel?

File: ruby.html.markdown [hunk 1]
```diff
@@ -576,6 +576,18 @@ Something.new.bar # => NoMethodError: undefined method `bar'
 Something.new.qux # => 'qux'
 ```
 
+# Sometimes you don't need a class, but still want to structure your data and have handy helper methods
+# In this case, you can use `Struct`.
+
+User = Struct.new(:first_name, :last_name, :user_name, :email_address) #=> User
```
Comment (vendethiel) [inline ruby.html.markdown]: I'd like it a lot if you could add an example with a block :)

**Comment by vendethiel** [comment]
@pzula any update? :)

**Comment by vendethiel** [comment]
Going to close this one for now :). Feel free to ping us here if you find some time working on it again! Thanks.

**Pull request closed without merging**